Doc/library/sqlite3.rst

```diff
@@ -1980,6 +1980,12 @@ Here's an example of both styles:
 
    [('C', 1972)]
 
+.. deprecated-removed:: 3.12 3.14
+
+   If named placeholders are used and *parameters* is a sequence,
+   :exc:`DeprecationWarning` is raised.
+   Starting with Python 3.14, :exc:`ProgrammingError` will be raised.
+
```

Comment:
Actually, this is the wrong place for `deprecated-removed`; this is a how-to, not a reference. We should move this to the _reference_ for `execute` and `executemany`.

Reply:
... and then as `versionchanged`, not `deprecated-removed` `.